tests/test_common.py

```diff
@@ -29,3 +33,30 @@ def test_handles_acronym():
 def test_handles_empty_string():
     assert decamelize(None) == ""
     assert decamelize("") == ""
+
+def test_py_samples_clientlib():
+    path_to_gen = MOCK / "client_library"
+    os.chdir(path_to_gen)
+    os.system("python3 -m synthtool")
```

Comment:
take a look and see if this can be replaced with directly calling `py_samples`: https://github.com/googleapis/synthtool/blob/master/tests/test_templates.py#L111-L123